.github/workflows/clippy.yml

```diff
@@ -35,18 +35,11 @@ jobs:
       with:
         github_token: "${{ secrets.github_token }}"
 
-    - name: rust-toolchain
-      uses: actions-rs/toolchain@v1.0.6
-      with:
-        toolchain: nightly
-        target: x86_64-unknown-linux-gnu
-        profile: minimal
-
     - name: Checkout
       uses: actions/checkout@v2.3.3
 
-    - name: Run cargo update
-      run: cargo update
+    - name: Install toolchain
+      run: rustup show active-toolchain
```

Comment:
If we remove the `rust-toolchain` step, from where does the `rustup` used in this command come from?

Reply:
GHA ships with stable rust+rustup already installed. See [GHA documentation about the pre-installed software](https://docs.github.com/en/free-pro-team@latest/actions/reference/specifications-for-github-hosted-runners#supported-software)

Reply:
That's convenient :)

BTW thanks for the work on this, you saved me another night fighting the CI :smile:

Reply:
NP, I like CI pain :smile: 

So can we merge this? :)

Reply:
Sure! I was hesitant about approving a PR I opened :smile: